app/controllers/SecureCommsAlertController.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import java.text.SimpleDateFormat
+
+import javax.inject.{Inject, Singleton}
+import models.SecureCommsDataModel
+import org.joda.time.DateTime
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent}
+import uk.gov.hmrc.play.bootstrap.controller.BaseController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SecureCommsAlertController @Inject()(implicit ec: ExecutionContext) extends BaseController {
```

Comment:
I don't think this controller is needed. For API#2 data we will have stub data in the acceptance/performance tests which populate the dynamic stub. Each piece of data has a route where it can be retrieved so we don't need to set up controllers or static routes for it.